spyder/plugins/findinfiles/plugin.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, parent=None):
         more_options = self.get_option('more_options')
         case_sensitive = self.get_option('case_sensitive')
         path_history = self.get_option('path_history', [])
+        search_in_index = self.get_option('search_in_index', 0)

```

Comment:
Please change this to

    search_in_index = self.get_option('search_in_index', default=0)